content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown

```diff
@@ -262,6 +296,9 @@ mtcars %>%
 The lambda function must return a data frame. `r link("group_map")` row binds the data 
 frames, recycles the grouping columns and structures the result as a grouped tibble. 
 
+>  `group_walk()` can be used when iterating on the groups is only desired for side effects. 
```

Comment:
I think if you're referring to the same function here, you can use "It" in lieu of repeating the function name.